app/templates/style_studio.html

```
@@ -5,335 +5,690 @@
+                                <span class="position-absolute top-0 start-100 translate-middle badge rounded-circle bg-dark border border-secondary p-1">
+                                    <i class="bi bi-plus text-yellow" style="font-size: 0.6rem;"></i>
+                                </span>
+                            </div>
                         </div>
+                        <h5 class="fw-bold mb-2">Drag & drop your selfie</h5>
+                        <p class="text-secondary text-sm mb-4">PNG, JPG, WebP up to 10MB</p>
+                        <button class="btn btn-light fw-bold px-4 py-2 rounded-3" type="button">Browse Files</button>
+                    </div>
+                    
+                    <div id="image-preview-container" class="d-none w-100 h-100 p-2 position-relative">
+                        <img id="image-preview" src="" class="img-fluid rounded-4 w-100 h-100" style="object-fit: cover; max-height: 500px;">
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)